### DISCUSSION TITLE
how to build for Pico devices

### DISCUSSION THREAD

**Discussion by dtboy1995** [icosa-gallery/open-brush#489] [Q&A] [answered]
![image](https://github.com/icosa-gallery/open-brush/assets/19227547/557105a7-0449-44d4-927d-03e300801e57)
How to build for Pico devices? Thank you

**Comment by mikeskydev** [comment, marked as answer]
Hi @dtboy1995, you can enable this by adding `PICO_SUPPORTED` to your android scripting define symbols.
We highly recommend making use of the built-in github actions builder pipeline which can automate all of this for you!

**Reply by dtboy1995** [reply]
thanks

**Reply by qweryty** [reply]
Are there any additional steps? Just defining `PICO_SUPPORTED` doesn't work for me.

**Reply by kris-w-dev** [reply]
hi, How to use the built-in github action builder pipeline for pico builds ?  I haven't found the specific steps to implement it yet.

**Reply by mikeage** [reply]
The default build job creates pico builds as part of the matrix. (if you have any followup questions, please create a separate discussion)
![Screenshot 2024-07-11 at 12 07 38](https://github.com/icosa-foundation/open-brush/assets/87874/e04c42e7-4c6b-43df-9d4d-512dda2600a3)
